### ISSUE TITLE
[frio] prevent double likes from the same user on the same post

### ISSUE THREAD

**Issue by foss-** [friendica/friendica#13735]
### Bug Description

In `This is Friendica, version 2023.09-rc that is running at the web location https://pirati.ca. The database version is 1542/1541, the post update version is 1507/1507.` it is possible to double like a post.

### Steps to Reproduce

1.  click `thumbs up` on a post
2.   click `thumbs up` on the same post

Actual Result:

In rare cases (depending on server load of own and remote instance) and if stars align, it is possible to double like the same post. This results in a situation where, while being able to `unlike` the post, that results in a situation with still one like by the user but an inactive thumbs up icon and now way of removing the remaining like 😣

Expected Result:

Ensure that likes are ever limited to one like per user per post or comment.

**Comment by MrPetovan** [comment]
When you reload the post (or open it in a separate tab), is the like display consistent with the last wrong state?

**Comment by foss-** [comment]
Yeah it is consistent over tabs and seems stuck in that unexpected state for that specific post so at least it is consistently wrong and not a caching issue.

**Comment by foss-** [comment]
Have not seen this in a while, closing.
